readme.md

```diff
@@ -213,36 +249,46 @@ In our `app/views/artists/index.html.erb`
 
 Now when we visit `/artists` in the browser, we see a list (index) of all artists. We have included links to the new page, and for the show page, but we haven't created those yet so....
 
-### You-Do: Show and New Actions (15/70)
-- Define `show`, and `new` controller actions for artists
-- Create `show`, and `new` views for artists
-- Display relevant info for an artist when you visit their show page
-- When you visit the `new` page, the relevant form should be displayed
-- Don't worry about submitting the form just yet
+### You-Do: Show and New Actions (15 / 85)
 
-## BREAK (10/80)
+- Define `show`, and `new` controller actions for `artists`
+- Create `show`, and `new` views for `artists`
+- Display relevant info for an `artist` when you visit their `show` page
```

Comment:
I would tell them which properties to show, but give them the freedom of how they're going to be displayed.